src/main/java/uk/gov/homeoffice/digital/sas/balancecalculator/module/NightHoursAccrualModule.java

```diff
@@ -12,10 +12,21 @@
 
 public class NightHoursAccrualModule extends AccrualModule {
 
+  private static final int MIDNIGHT = 0;
```

Comment:
My fault on this, I was meaning something like
NIGHT_HOURS_START=23
NIGHT_HOURS_END=6
Then if they ever changed to 10pm-5am we'd only need to change the constant value not it's name